package.json

```diff
@@ -59,7 +59,7 @@
   },
   "scripts": {
     "start-local": "./scripts/serve-local.sh",
-    "start-joplin-cloud": "NODE_PATH=./src CMS_API='http://joplin.herokuapp.com/api/graphql' CMS_MEDIA='https://joplin-austin-gov.s3.amazonaws.com/media' npm-run-all -p watch-css start-js",
```

Comment:
Lets change this back to default joplin prod before merge/maybe we should make it easier to run this command off of different branches